InstagramKit/Models/InstagramUser.m

```diff
@@ -48,7 +48,7 @@ - (instancetype)initWithInfo:(NSDictionary *)info
 - (void)updateDetailsWithInfo:(NSDictionary *)info
 {
     self.username = [[NSString alloc] initWithString:info[kUsername]];
-    self.fullName = [[NSString alloc] initWithString:info[kFullName]];
+    self.fullName = (IKNotNull(info[kProfilePictureURL])) ? [[NSString alloc] initWithString:info[kFullName]] : nil;
```

Comment:
shouldn't it be like;
`self.fullName =IKNotNull(info[kFullName]) ? [[NSString alloc] initWithString:info[kFullName]] : nil;` ??

Reply:
Happened to me as well, please merge someone

Reply:
@stremsdoerfer thanks for the PR. You need to change `kProfilePictureURL` to `kFullName` as @ardramli pointed out.